src/storage/mvcc/txn.rs

```diff
@@ -218,13 +230,23 @@ impl<'a, S: Snapshot + ?Sized> MvccSnapshot<'a, S> {
             }
         }
         // Find the latest write below our start timestamp.
-        match meta.iter_items().find(|x| x.get_commit_ts() <= ts) {
-            Some(x) => {
+        if let Some(x) = first_meta.iter_items().find(|x| x.get_commit_ts() <= ts) {
```

Comment:
@BusyJay Tried it already. However the problem is that the first_meta is borrowed but others are load inside the loop (owned). I just failed to integrate them together.